docs/pages/dev/docs.md

```diff
@@ -13,9 +13,9 @@ to add it to the navigation in that file. Styles are defined in `global_style.cs
 
 If you only want to build the docs as they are, from the top level of the repository, run
 `make docs`. The documentation will be built and stored in `docs/out`. The output of this is
-suitable to be served using a simple HTTP server, such as `npx serve` or Github Pages.
+suitable to be served using a simple HTTP server, such as `npx serve` or GitHub Pages.
 
 However, if you plan to write and edit documentation, there are better tools available. From the
-`docs/` directory, runt he command `npm run dev`, which will run a development server that live
+`docs/` directory, run the command `npm run dev`, which will run a development server that live
```

Comment:
TL;DR: The "run the" was the real typo fix, the rest are just consistency/copyright nits.